71_RayTracingPipeline/app_resources/raytrace_shadow_triangle.rchit.hlsl

```diff
@@ -0,0 +1,7 @@
+#include "common.hlsl"
+
+[shader("closesthit")]
+void main(inout OcclusionPayload payload, in BuiltInTriangleIntersectionAttributes attribs)
+{
+    payload.attenuation = 0;
+}
```

Comment:
theoretically you could trace the shadow ray  with both `TerminateOnFirstHitKHR ` and the `SkipClosestHitShaderKHR` flags then you don't need a closest hit shader for the shadow rays at all

Reply:
or would you, because unlike SER and ray-query you have no way to know anything about the ray after the intrinsic returns?

Reply:
I see in the review I told you to do something else, I'll play with it myself when merging